content/changelog/2025/10-08-postgresql-18.md

```diff
@@ -0,0 +1,20 @@
+---
+title: PostgreSQL 18 is available on Clever Cloud
+description: And there are lots of great new features still to come!
+date: 2025-10-08
+tags:
+  - addons
+  - update
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+excludeSearch: true
+---
+
+[PostgreSQL 18](https://www.postgresql.org/about/news/postgresql-18-released-3142/) is available on Clever Cloud, with the same default extension enabled for PostgreSQL 17. This release comes with `io_uring` for asynchronous I/O, better performance, [virtual generated columns](https://www.postgresql.org/docs/18/sql-createtable.html#SQL-CREATETABLE-PARMS-GENERATED-STORED), UUIDv7 generation, improved text processing and multiple authentication and security features.
+
+You can simply test it with a new add-on or update an existing through automatic and safe Clever Cloud's migration process.
```

Comment:
You can simply test it with a new add-on or update an existing one through automatic migration and safe Clever Cloud's migration process.